dipy/core/optimize.py

```diff
@@ -433,12 +433,12 @@ def __init__(self, m, *, A=None, L=None):
         .. footbibliography::
         """
         # Input
-        self.A = A
-        self.L = L
+        self.A = A  # list: [zeros, H(omega), L(alpha)] --- see eq 22
```

Comment:
This is eq 22 in DelaHaije2020?

Reply:
Yes, I will put a comment in somewhere that makes this clear